### ISSUE TITLE
EnsembleSummary plots can ignore idxs

### ISSUE THREAD

**Issue by isaacsas** [SciML/SciMLBase.jl#364]
```julia
using Catalyst, DifferentialEquations, Plots
rn = @reaction_network begin
       1.0, A --> 0
       2.0, B --> 0
       4.0, C --> 0
       end
dprob = DiscreteProblem(rn, [:A => 100, :B => 10, :C => 5], (0.0,20.0))
jprob = JumpProblem(rn, dprob, Direct())
eprob = EnsembleProblem(jprob)
sims = solve(eprob, SSAStepper(), trajectories=100)
@unpack A,C = rn
plot(sims, idxs=[A,C])
```
plots just the trajectories for species `A` and `C`, while 
```julia
summ = EnsembleSummary(sims)
plot(summ; idxs=[A,C])
```
plots all three species.

**Comment by cfrontin** [comment]
i have a similar issue! i mine is similar but i don't have dictionary output, it fails with just a vector output

**Comment by natema** [comment]
Similar issue here. 
Looking at [the plot recipe for EnsembleSummary](https://github.com/SciML/SciMLBase.jl/blob/d7a51f7f1b58f01cfc3a9bb1145dab2e50323aa8/src/ensemble/ensemble_solutions.jl#L147-L199), I don't see the `idxs` keyword described in the documentation, but rather a `trajectories` keyword to which one can provide the indices of the desired variables. 
So, in @isaacsas' case, I'd try `plot(summ; idxs=(1,3))`.

On top of that, in my case I have an ODESystem preprocessed by `structural_simplify` (MTK), and I would like to plot a variable of `summ` which is in `sims.prob.f.paramsyms` but not in `sims.prob.f.syms` (using the variable names of @isaacsas' example).  I wonder if there's an easy way to do that.

**Comment by ChrisRackauckas** [comment]
Yeah this is just a bug in the plot recipe. Right now the answer is to plot each individually until someone gets around to handling it.

**Comment by natema** [comment]
Just to suggest an alternative to each plotting individually, for large systems. 
AFAIS, the order of the variable is preserved by `solve` and `EnsembleSummary`, so one way to retrieve the corresponding trajectory indices is with `println.(enumerate(states(sys))` where `sys` is the `ODESystem`.

**Comment by ChrisRackauckas** [comment]
That works for variables other than the ones that are just the observed
